sql/core/src/test/resources/sql-tests/results/comments.sql.out

```diff
@@ -132,20 +132,9 @@ select 1 as a
 struct<>
 -- !query output
 org.apache.spark.sql.catalyst.parser.ParseException
-
-Unclosed bracketed comment(line 3, pos 0)
-
-== SQL ==
-/*abc*/
-select 1 as a
-/*
-^^^
-
-2 as b
-/*abc*/
-, 3 as c
-
-/**/
+{
+  "errorClass" : "_LEGACY_ERROR_TEMP_055"
```

Comment:
Did we loose the context?

Reply:
Yes. We don't provide context in many places. This is one of them.

Reply:
Just in case, users will see the same error message by default (not JSON).